tools/wptrunner/wptrunner/executors/executorselenium.py

```diff
@@ -102,29 +103,34 @@ def close_old_windows(self):
         self.webdriver.switch_to_window(self.runner_handle)
         return self.runner_handle
 
-    def get_test_window(self, window_id, parent):
+    def get_test_window(self, window_id, parent, timeout=5):
```

Comment:
Can you add a comment for all the executors which now have a timeout here explaining *why*?